web/src/app/shared/components/nav-bar/nav-bar.component.html

```diff
@@ -1,78 +1,48 @@
-<nav class="navbar main navbar-fixed-top">
-  <div class="navbar  no-border-radius navbar-inverse no-gutter col-xs-12">
-    <div class="pull-left col-sm-3 col-md-4">
-      <button type="button" *ngIf="!showNavbar" class="btn btn-default navbar-toggle show pull-left" (click)="toggleNavBar();"
-        aria-expanded="true" aria-controls="navbar" data-toggle="tooltip" title="Menu">
-        <span class="material-icons md-24 white">menu</span>
+<mat-sidenav-container class="example-container">
```

Comment:
please use BEM methodology for class name. EG sidenav, child components: sidenav__logout-button, sidenav__user-info